src/table/body-cell/td-element.tsx

```diff
@@ -20,55 +22,63 @@ export interface TableTdElementProps {
   stripedRows?: boolean;
   hasSelection?: boolean;
   hasFooter?: boolean;
-  isVisualRefresh?: boolean;
+  getStickyColumnProperties: () => GetStickyColumnProperties;
+  tdRef: React.Ref<HTMLTableCellElement>;
```

Comment:
Should we make this optional?

Reply:
No strong opinion here, but I'd prefer keeping it required. Unless this component is used outside of our table, this prop should be there.